backend/src/packages/articles/article.repository.ts

```diff
@@ -40,6 +40,24 @@ class ArticleRepository implements IRepository {
     return ArticleEntity.initialize(article);
   }
 
+  public async findArticleWithAuthor({
+    id,
+  }: {
+    userId?: number;
+    id: number;
+  }): Promise<ArticleEntity | null> {
+    const article = await this.articleModel
+      .query()
+      .findById(id)
+      .withGraphJoined(this.defaultRelationExpression);
+
+    if (!article) {
+      return null;
+    }
+
+    return ArticleEntity.initializeWithAuthor(article);
+  }
```

Comment:
As conflicts with `development` would be resolved - I guess this method won't be needed as you may use default `find` that was implemented